main/ertp/api/amount-math.md

```diff
@@ -1,189 +1,304 @@
 
 ```js
-const { amountMath } = makeIssuerKit('fungible');
-const empty = amountMath.getEmpty();
-const fungible1 = amountMath.make(1);
+const { amountMath: quatloosAmountMath } = makeIssuerKit('quatloos');
+const empty = quatloosAmountMath.getEmpty();
+const quatloos1 = quatloosAmountMath.make(1);
 
 // returns true
-amountMath.isEmpty(empty)
+quatloosAmountMath.isEmpty(empty)
 
 // returns false
-amountMath.isEmpty(fungible1)
+quatloosAmountMath.isEmpty(quatloos1)
 ```
 
 ## amountMath.isGTE(leftAmount, rightAmount)
 - `leftAmount` `{Amount}`
 - `rightAmount` `{Amount}`
 - Returns: `{boolean}`
 
-Returns true if the leftAmount is greater than or equal to the rightAmount. For non-scalars, "greater than or equal to" depends on the kind of amount, as defined by the MathHelpers. For example, whether rectangle A is greater than rectangle B depends on whether rectangle A includes rectangle B as defined by the logic in MathHelpers.
+Returns `true` if the `value` of `leftAmount` is greater than or equal to
+the `value` of `rightAmount`. Both `amount` arguments must have the same
+`brand` as this `amountMath`.
+
+For non-fungible `values`, what "greater than or equal to" is depends on the 
+kind of `amountMath`. For example, { 'seat 1', 'seat 2' } is considered
+greater than { 'seat 2' } because the former both contains all of the latter's 
+contents and has more elements.
 
 ```js
-const { amountMath } = makeIssuerKit('fungible');
-const empty = amountMath.getEmpty();
-const fungible1 = amountMath.make(1);
+const { amountMath: quatloosAmountMath } = makeIssuerKit('quatloos');
+const empty = quatloosAmountMath.getEmpty();
+const quatloos5 = quatloosAmountMath.make(5);
+const quatloos10 = quatloosAmountMath.make(10);
 
 // Returns true
-amountMath.isGTE(fungible1, empty);
-
+quatloosAmountMath.isGTE(quatloos5, empty);
+// Returns false
+quatloosAmountMath.isGTE(empty, quatloos5);
+// Returns true
+quatloosAmountMath.isGTE(quatloos10, quatloos5);
 // Returns false
-amountMath.isGTE(empty, fungible1);
+quatloosAmountMath.isGTE(quatloos5, quatloos10);
+// Returns true
+quatloosAmountMath.isGTE(quatloos5, quatloos5);
 ```
 
 ## amountMath.isEqual(leftAmount, rightAmount)
 - `leftAmount` `{Amount}`
 - `rightAmount` `{Amount}`
 - Returns: `{boolean}`
 
-Returns true if the leftAmount equals the rightAmount. We assume that if isGTE is true in both directions, isEqual is also true.
+Returns `true` if the `value` of `leftAmount` is equal to
+the `value` of `rightAmount`. Both `amount` arguments must have the same
+`brand`.
+
+For non-fungible `values`, "equal to" depends on the kind of `amountMath`. 
+For example, { 'seat 1', 'seat 2' } is considered
+unequal to { 'seat 2' } because the number of items in the former is
+different from that of the latter. Similarly { `seat 1` } and { `seat2` } 
+are considered unequal because the conthe former both contains all of the latter's 
+contents and has more elements.
```

Comment:
```suggestion
different from that of the latter. Similarly { `seat 1`,  `seat2`  } and { `seat2` } 
are considered unequal because the former has elements that are not contained in the latter.
```